pennylane/transforms/metric_tensor.py

```diff
@@ -80,6 +90,31 @@ def metric_tensor(tape, approx="block-diag", diag_approx=None, allow_nonunitary=
         func: Function which accepts the same arguments as the QNode. When called, this
         function will return the metric tensor.
 
+    The block-diagonal part of the metric tensor always is computed using the
+    covariance-based approach. If no approximation is selected,
+    the off block-diagonal is computed using Hadamard tests.
+
+    .. warning::
+
+        Performing the Hadamard tests requires a device
+        that has an additional wire as compared to the wires on which the
+        original circuit was defined. This wire may be specified via ``aux_wire``.
+        By default, contiguous wire numbering and usage is assumed and the additional
+        wire is set to the next wire of the device after the circuit wires.
+        If the given or inferred ``aux_wire`` does not exist on the device,
+        a warning is raised and the block-diagonal approximation is computed instead.
+        It is significantly cheaper in this case to explicitly set ``approx="block-diag"`` .

```

Comment:
```suggestion
        Performing the Hadamard tests requires a device
        that has an additional wire as compared to the wires on which the
        original circuit was defined. This wire may be specified via ``aux_wire``.

        By default, contiguous wire numbering and usage is assumed and the additional
        wire is set to the next wire of the device after the circuit wires.

        If the given or inferred ``aux_wire`` does not exist on the device,
        a warning is raised and the block-diagonal approximation is computed instead.
        It is significantly cheaper in this case to explicitly set ``approx="block-diag"`` .
```
line breaks for readability 🙂